charts/gateway-helm/values.tmpl.yaml

```diff
@@ -77,6 +77,10 @@ certgen:
   job:
     annotations: {}
     resources: {}
+    pod:
```

Comment:
do we need the top level `pod` field or can we do w/o it ? @envoyproxy/gateway-maintainers

Reply:
hey @ncsham lets rm `pod` from here, since its not an explicit field in the job api and not needed here in this API

Reply:
sure , will do

Reply:
this is done